packages/app-store/apps.server.generated.ts

```diff
@@ -12,6 +12,7 @@ const googlecalendar_api = import("./googlecalendar/api");
 const hubspotothercalendar_api = import("./hubspotothercalendar/api");
 const huddle01video_api = import("./huddle01video/api");
 const jitsivideo_api = import("./jitsivideo/api");
+const larkcalendar_api = import("./larkcalendar/api");
```

Comment:
This file as well shouldn't be pushed?

Reply:
same here @zomars ?

Reply:
I told @hariombalhara  that we want this tracked now. It is auto-generated of commit. Some users complained that on build time some times it failed and we don't want to generate this file on each field. Also this file rarely changes so we may just track it.